mqtt_bridge/app.py

```diff
@@ -0,0 +1,100 @@
+import inject
+import paho.mqtt.client as mqtt
+import rclpy
+from rclpy.node import Node
+
+from .bridge import create_bridge
+from .mqtt_client import create_private_path_extractor
+from .util import lookup_object
+
+
+def create_config(mqtt_client, serializer, deserializer, mqtt_private_path):
+    if isinstance(serializer, str):
+        serializer = lookup_object(serializer)
+    if isinstance(deserializer, str):
+        deserializer = lookup_object(deserializer)
```

Comment:
```suggestion
        deserializer = lookup_object(deserializer.value)
```

Reply:
Yeah the changes makes sense!! I forgot using the .value for serializer and deserializer params, thanks for the suggestions